mmv1/templates/terraform/examples/uptime_check_config_status_code.tf.erb

```diff
@@ -0,0 +1,36 @@
+resource "google_monitoring_uptime_check_config" "<%= ctx[:primary_resource_id] %>" {
+  display_name = "<%= ctx[:vars]["display_name"] %>"
+  timeout      = "60s"
+
+  http_check {
+    path = "some-path"
+    port = "8010"
+    request_method = "POST"
+    content_type = "URL_ENCODED"
+    body = "Zm9vJTI1M0RiYXI="
+
+    accepted_response_status_codes {
```

Comment:
could your example have an additional status codes block

Reply:
added another exampl

Reply:
Sorry what I meant here is that this is an array. So you should be able to provide multiple blocks inline for multiple types of accepted status blocks.

For example 
```hcl
resource "google_monitoring_uptime_check_config" "-" {  
...
    accepted_response_status_codes {
      status_class = "STATUS_CLASS_2XX"
    }
    accepted_response_status_codes {
      status_class = "STATUS_CLASS_3XX"
    }
...
}
```

or even 
```hcl
resource "google_monitoring_uptime_check_config" "-" {  
...
    accepted_response_status_codes {
      status_class = "STATUS_CLASS_2XX"
    }
    accepted_response_status_codes {
            status_value = 301
    }
    accepted_response_status_codes {
            status_value = 302
    }
...
}
```

Reply:
gotcha, makes sense.

Reply:
done + tested